guides/common/assembly_installing-satellite-server-disconnected.adoc

```diff
@@ -22,6 +22,26 @@ include::modules/proc_downloading-the-binary-dvd-images.adoc[leveloffset=+1]
 
 include::modules/proc_configuring-the-base-operating-system-with-offline-repositories.adoc[leveloffset=+1]
 
+ifdef::satellite[]
```

Comment:
Instead of duplicating content, please have a look at using a generic name in files and the key of an attribute, and then set the value of the attribute in https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc.

Refs https://github.com/theforeman/foreman-documentation/pull/3785#discussion_r2070693691

In principle, this could be similar to the whole Web Console/Cockpit situation: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L58